### ISSUE TITLE
Master branch does not build

### ISSUE THREAD

**Issue by leynebe** [stripe/stripe-mirakl-connector#27]
Hello!

Since 1 or 2 days the build on master branch doesn't seem to work anymore.

`cp .env.dist .env`
`cd examples/docker`
`docker-compose up`

```
Step 14/18 : RUN composer install --prefer-dist --no-scripts --no-dev --no-autoloader -q     && rm -rf /root/.composer
 ---> Running in 52b29627be6d
Your lock file does not contain a compatible set of packages. Please run composer update.
ERROR: Service 'app' failed to build: The command '/bin/sh -c composer install --prefer-dist --no-scripts --no-dev --no-autoloader -q     && rm -rf /root/.composer' returned a non-zero code: 2
```

When adding the step `RUN composer update`, I get the following output:

```
Step 14/19 : RUN composer update

...

  Problem 1
    - Root composer.json requires phpstan/phpstan-symfony ^0.11.6 -> satisfiable by phpstan/phpstan-symfony[0.11.6].
    - phpstan/phpstan-symfony 0.11.6 requires ext-simplexml * -> it is missing from your system. Install or enable PHP's simplexml extension.
  Problem 2
    - php-coveralls/php-coveralls[v2.2.0, ..., v2.4.2] require ext-simplexml * -> it is missing from your system. Install or enable PHP's simplexml extension.
    - Root composer.json requires php-coveralls/php-coveralls ^2.2 -> satisfiable by php-coveralls/php-coveralls[v2.2.0, ..., v2.4.2].

...
```

I'm not proficient enough with php/composer to solve it or I would have done it myself.

Thanks

Ben

**Comment by youssef-stripe** [comment]
Hello!

Thank you for flagging. This looks related to the release of Composer 2 a few days ago. We will fix this as soon as possible.

**Comment by youssef-stripe** [comment]
This is fixed in v1.2.1.

Thanks again for flagging and have a great day!
